actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -693,6 +693,22 @@ def transform_keys!(&block)
       self
     end
 
+    # Returns a new <tt>ActionController::Parameters</tt> instance with the
+    # results of running +block+ once for every key. This method includes the
+    # keys of nested hashes and array of hashes. The values are unchanged.
```

Comment:
The grammatically correct phrasing here would be "arrays of hashes".

I think I prefer the version used in the main `deep_transform_keys` docs though: "nested hashes and arrays of hashes" implies to me that only one level of nesting is covered, and the keys of a hash within an array-of-arrays wouldn't be transformed, for example, when in fact they would.